.github/workflows/ios.yml

```
@@ -12,14 +12,22 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: 'install all ruby dependencies (e.g: Fastlane)'
-        run: bundle install
+      - name: install danger-lin
```

Looks like the step's name was misspelled.
```suggestion
      - name: install danger-lint
```